terraform/modules/public-load-balancer/main.tf

```diff
@@ -16,6 +23,13 @@ resource "aws_lb_target_group" "public" {
   health_check {
     path = var.health_check_path
   }
+  
+  tags = merge(
+    var.additional_tags,
+    {
+      name      = "${var.app_name}-tg"
```

Comment:
it doesn't have the workspace interpolated in comparison to the previous tags above and name of the resource too